docs/getting-started/getting-started-with-docker.rst

```diff
@@ -239,7 +239,7 @@ from the 1000 Genomes project:
     $> /usr/local/bin/bamstats 4 NA12878.chrom20.ILLUMINA.bwa.CEU.low_coverage.20121211.bam
 
 What's really going on here? The ``bamstats`` command above is a simple
-script I wrote to make it easier to call BAMStats. This is what I used
+script we wrote to make it easier to call BAMStats. This is what we used
```

Comment:
Sentence reads a bit off:

This is what we used the ``COPY`` command to move into the Docker image via the Dockerfile.

Reply:
Rewrote this heavily for clarity; turns out it references an earlier script.